src/TorchSharp/TorchSharp.csproj

```diff
@@ -22,7 +22,8 @@
 
   <ItemGroup>
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
-    <PackageReference Include="System.Memory" Version="4.5.3" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.1" />
```

Comment:
>SixLabors [](http://example.com/codeflow?start=31&length=9)

@NiklasGustafsson ImageSharp library is under Apache 2.0 license. Do we need to validate anything here before using the library?

Reply:
For a package dependency? I don't think so. It would be different if we used their source code, I think.

Reply:
That is fine then. I was wondering because I am seeing the following in the NuGet package page

![image](https://user-images.githubusercontent.com/10833894/170353234-535b7082-3b3f-464d-ae72-e682fc0d719f.png)

Reply:
I don't think it's a licensing issue, but we may want to think about whether actual implementations (not the interfaces) of image I/O belong in a different NuGet package.

Reply:
I know @michaelgsharp has been looking at this for ML.NET. Michael, could you comment / weigh in?

Reply:
We are planning to do some work in ML.NET to use ImageSharp but we haven't gotten into that yet.

Reply:
If we were taking a dependency on a commercial library, that would be one thing. This one has both an OSS license and a commercial one, so we're not forcing users of TorchSharp to go with a commercial license.